File: runit_server/tools/javascript/runner.js
const args = process.argv

let functionArguments;

try {
    if (args.length > 3) {
        const filename = args[2]
        const functionname = args[3]
        
        const method = require(filename)[functionname]
    
        if (args.length > 4) functionArguments = args[4]
    
        if (functionArguments !== undefined) {
            method(functionArguments)
        } else {
            method()
        }
    
    }
} catch (error) {
    console.log(error)
}

File: runit_server/tools/python/runner.py
import os
import sys
import inspect

args = sys.argv
functionArguments = None

try:
    if len(args) >= 3:
        filename = args[1]
        functionname = args[2]

        sys.path.append(os.path.abspath(os.curdir))
        module = __import__(inspect.getmodulename(filename))
        method = [f[1] for f in inspect.getmembers(module, inspect.isfunction) if f[0] == functionname][0]

        if len(args) > 3:
            functionArguments = args[3]

        if functionArguments is not None:
            method(functionArguments)
        else:
            method()
    
except Exception as e:
    print(str(e))